specs/linux/ipmi_lan.table

```diff
@@ -0,0 +1,19 @@
+table_name("ipmi_lan")
+description("IPMI LAN information.")
+schema([
+    Column("mc_name", TEXT, "Name of the MC"),
+    Column("mc_id", TEXT, "Device ID of the MC"),
```

Comment:
I wanted to be specific here that those values were the name and if of the MC, and not the LAN, since an `id` column in `ipmi_lan` would, IMO, imply that the `id` will be the LAN `id`.  What do you think?
 
Actually with that said, I now think the `device_id` column in `ipmi_mc` should just be `id`.  Thoughts?